lust/lib/pages/rootPage.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/cupertino.dart';
+import 'package:lust/utils/autenthicationAPI.dart';
+import 'capacityPage.dart';
+import 'loginPage.dart';
+
+class RootPage extends StatefulWidget {
+  RootPage({this.auth});
+
+  final BaseAuth auth;
+
+  @override
+  _RootPageState createState() => _RootPageState();
+}
+
+enum LogInOut { SIGNEDIN, NOTSIGNEDIN}
+
+class _RootPageState extends State<RootPage> {
+  LogInOut authStatus = LogInOut.NOTSIGNEDIN;
+
+  _RootPageState();
+
+  @override
+  initState() {
```

Comment:
what is the purpose of this function?
it provokes an error message without doing anything usefull